src/data/graphql/queries.graphql

```diff
@@ -303,6 +303,18 @@ query RecoveryRolesUsers($colonyAddress: String!, $endBlockNumber: Int) {
   }
 }
 
+query ArchitectureRoleUsers($colonyAddress: String!) {
+  architectureRoleUsers(colonyAddress: $colonyAddress) @client {
+    id
+    profile {
+      avatarHash
+      displayName
+      username
```

Comment:
This is so f****** stupid!!!!!

This doesn't work because of `username` and I have no idea why!!!!!!

(there are a couple of more issues, but the main problem is `username`)

If you change this to anything other than, `username` it will work.

This is how I've changed it:
```suggestion
      userName
```